core/src/main/java/org/apache/accumulo/core/security/crypto/NoEncryptionStrategy.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with this
+ * work for additional information regarding copyright ownership. The ASF
+ * licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.accumulo.core.security.crypto;
+
+import java.io.InputStream;
+import java.io.OutputStream;
+
+import org.apache.accumulo.core.conf.AccumuloConfiguration;
+
+/**
+ * The default encryption strategy which does nothing.
+ */
+public class NoEncryptionStrategy implements EncryptionStrategy {
+
+  @Override
+  public boolean init(Scope encryptionScope, AccumuloConfiguration conf) {
+    return true;
```

Comment:
What's this boolean for? Should probably succeed (void) or fail (throw), but not succeed with `false` return code... unless it's used for something else?

Reply:
I have code in the factory that returns a NoEncryptionStrategy when init returns false.  I guess false is more of a fail quietly and throw exception is fail loudly.

Reply:
I don't think we should ever fail quietly when it comes to encryption. :smiley_cat:

Reply:
Haha fair enough.  I will make it return void.